middleware.js

```diff
@@ -49,6 +61,24 @@ module.exports.fetchRandomGameData = async (req, res, next) => {
   }
 };
 
+module.exports.fetchRandomGameDataArr = async (req, res, next) => {
```

Comment:
Wait, aren't isn't the app already fetching a single random data in the function above? Maybe the function above could be reused.

Reply:
I was planning to do exactly that initially, but then realized that the function above quite different and is necessary on login and register pages. I might refactor both functions a bit once i get to adding error handling to `fetchRandomGameDataArr` function

Reply:
I'll give it a pass this time. Make sure to refactor and unify these functions otherwise it's redundant repetition.